code/integration-tests/local-integration-tests/src/cross_chain_transfer.rs

```diff
@@ -450,42 +451,51 @@ fn transfer_to_sibling() {
 	});
 }
 
-/// if Alice sends amount of their tokens and these are above weight but less than ED,
-/// than our treasury takes that amount, sorry Alice
-/// from: Acala
 #[test]
-fn transfer_from_relay_chain_deposit_to_treasury_if_below_ed() {
+fn one_chain_cannot_print_relay_native_reserve_tokens_on_us() {
 	simtest();
 	let receiver = charlie();
-	let (picasso_treasury, under_ed) = This::execute_with(|| {
+
+	let original_bob_on_sibling = This::execute_with(|| {
 		use this_runtime::*;
 		let under_ed = under_existential_deposit::<AssetsRegistry>(LocalAssetId::KSM, 3);
-		assert_eq!(Tokens::free_balance(CurrencyId::KSM, &AccountId::from(receiver)), 0,);
-		(Tokens::free_balance(CurrencyId::KSM, &this_runtime::TreasuryAccount::get()), under_ed)
+		assert_ok!(Tokens::deposit(CurrencyId::KSM, &alice().into(), 100_000_000_000_000));
+		Tokens::free_balance(CurrencyId::KSM, &alice().into())
 	});
 
-	KusamaRelay::execute_with(|| {
-		use relay_runtime::*;
-		assert_ok!(XcmPallet::reserve_transfer_assets(
-			Origin::signed(ALICE.into()),
-			Box::new(Parachain(THIS_PARA_ID).into().into()),
-			Box::new(Junction::AccountId32 { id: receiver, network: NetworkId::Any }.into().into()),
-			Box::new((Here, under_ed).into()),
-			0
-		));
+	let alice_original = This::execute_with(|| {
+		use this_runtime::*;
+		assert_ok!(Tokens::deposit(CurrencyId::KSM, &alice().into(), 100_000_000_000_000));
+		Tokens::free_balance(CurrencyId::KSM, &alice().into())
 	});
 
+	let alice_from_amount = alice_original / 10;
+	let alice_remaining = alice_original - alice_from_amount;
+	let weight_to_pay = (alice_from_amount / 2) as u64;
```

Comment:
Why is it divided by 2?

Reply:
unsure - this was copied from #2118. @dzmitry-lahoda?

Reply:
Resolving to get this merged, we will review this later: https://app.clickup.com/t/39w9w06

Reply:
not all tests were migrated from resonable approximations to fee estimator.